scripts/qiita-cron-job

```diff
@@ -8,31 +8,55 @@
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
 
+import click
+
 from qiita_db.util import (
-    purge_filepaths, empty_trash_upload_folder, purge_files_from_filesystem)
+    purge_filepaths as qiita_purge_filepaths,
+    empty_trash_upload_folder as qiita_empty_trash_upload_folder,
+    purge_files_from_filesystem as qiita_purge_files_from_filesystem)
 from qiita_db.meta_util import (
-    update_redis_stats, generate_biom_and_metadata_release)
+    update_redis_stats as qiita_update_redis_stats,
+    generate_biom_and_metadata_release as
+    qiita_generate_biom_and_metadata_release)
 
 
-# This script will perform these jobs:
-# 1. purge_filepaths: remove files from that are leftover in the
-#    qiita.filepath and are present in the filesystem
-# 2. empty_trash_upload_folder: remove files that are present in the trash
-#    of the upload folders
-# 3. update_redis_stats: updates the redis stats information
-# 4. generate public releases of biom tables and metadata
-#
-# Note that is responsability of the Qiita install system admin to add to a
-# cron job this script and responsible to define how often it should run
+@click.group()
+def commands():
+    pass
+
+
+@commands.command()
+@click.option('--remove/--no-remove', default=True,
+              help=('remove files from that are leftover in the '
```

Comment:
Not sure what the help text in here means, would you mind rephrasing. Also, the parenthesis are not needed.